modules/oadp-release-notes-1-3-1.adoc

```diff
@@ -42,9 +42,9 @@ With this fix, `must-gather` now collects CRD's shipped by OADP and can use the
 
 link:https://issues.redhat.com/browse/OADP-3229[OADP-3229]
 
-.Garbage collection has the required default frequency
+.Garbage collection has the correct description for the default frequency
 
-Previously, the `garbage-collection-frequency` field had a wrong description for the default frequency. With this update, the `gc-controller` correctly executes its reconciliation loop every hour by default. As a result, garbage collection now has the expected default frequency.
+Previously, the `garbage-collection-frequency` field had a wrong description for the default frequency. With this update, the `gc-controller` correctly executes its reconciliation loop every hour by default. As a result, garbage collection now has the expected description for the default frequency.
```

Comment:
Previously, the `garbage-collection-frequency` field had a wrong description for the default frequency value. With this update,  'garbage-collection-frequency'   has the correct value of the  `gc-controller` reconciliation  default   frequency, which is one hour.